src/modules/rc_update/rc_update.cpp

```diff
@@ -215,8 +215,12 @@ void RCUpdate::parameters_updated()
 		const uint16_t throttle_min = _parameters.min[throttle_channel];
 		const uint16_t throttle_trim = _parameters.trim[throttle_channel];
 		const uint16_t throttle_max = _parameters.max[throttle_channel];
+		const bool throttle_rev = _parameters.rev[throttle_channel];
 
-		if (throttle_min == throttle_trim) {
+		const bool normal_case = !throttle_rev && (throttle_trim == throttle_min);
+		const bool reversed_case = throttle_rev && (throttle_trim == throttle_max);
```

Comment:
These cases are relevant only because of the hotfix in the QGC right? If so, should we note it in the code?

Codewise this is a nice way to wrap up https://github.com/PX4/PX4-Autopilot/pull/21576!

Reply:
Yes, that's written in the comment just above this logic:
https://github.com/PX4/PX4-Autopilot/pull/21682/files#diff-bc65e8936281291b7e9801b185ee1740a6f2d4207e3b539c059469daa6cdfcc9R210-R211